python/ql/lib/semmle/python/security/dataflow/LogInjectionCustomizations.qll

```diff
@@ -47,7 +47,35 @@ module LogInjection {
    * A logging operation, considered as a flow sink.
    */
   class LoggingAsSink extends Sink {
-    LoggingAsSink() { this = any(Logging write).getAnInput() }
+    LoggingAsSink() {
+      this = any(Logging write).getAnInput() and
+      // since the inner implementation of the `logging.Logger.warn` function is
+      // ```py
+      // class Logger
+      //     def warn(self, msg, *args, **kwargs):
+      //     warnings.warn("The 'warn' method is deprecated, "
+      //         "use 'warning' instead", DeprecationWarning, 2)
+      //     self.warning(msg, *args, **kwargs)
```

Comment:
```suggestion
      // class Logger:
      //     def warn(self, msg, *args, **kwargs):
      //         warnings.warn("The 'warn' method is deprecated, "
      //             "use 'warning' instead", DeprecationWarning, 2)
      //         self.warning(msg, *args, **kwargs)
```

Reply:
oh, I must have messed that up when I copy-pasted :see_no_evil: thanks :heart: